src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -3484,26 +3484,48 @@ public override Ast Copy()
         internal override string GetTooltip()
         {
             var sb = new StringBuilder();
-            if (IsStatic)
+            var classMembers = ((TypeDefinitionAst)Parent).Members;
+            for (int i = 0; i < classMembers.Count; i++)
             {
-                sb.Append("static ");
-            }
+                var methodMember = classMembers[i] as FunctionMemberAst;
+                if (methodMember is null ||
+                    !Name.Equals(methodMember.Name) ||
+                    IsStatic != methodMember.IsStatic)
```

Comment:
What the tooltip for non-default constructors looks like? For a PS class `A`, I expect it would be
```
[A] new(paramA)
[A] new(paramB)
```

Reply:
Using this test class:
```
class MyCustomType
{
    MyCustomType ([string] $s)
    {
    }
    MyCustomType ([int] $i)
    {
    }
    MyCustomType ([string] $s, [bool]$whocares)
    {
    }
}
```
It looks like this:
```
MyCustomType(string s)
MyCustomType(int i)
MyCustomType(string s, bool whocares)
```
which is the same you get when the type has been added and the tooltip isn't generated from the AST.